### PR TITLE
fix: Cleanup toolbar iframe views

### PR DESCRIPTION
**Pull request by ryan953**
Quick pass through to remove extra markup returned inside the iframe views that sentry-toolbar loads. 

Also, I moved the `setTimeout` call up inside the if-statement, so that its behavior aligns with the text on the page: if the window closes on it's own then everything is ok. If the window does not close then you want to read the page, and probably checkout the console. Right now the window closes on it's own no matter what so no debugging info is available.

- don't need a `style` attr inside a `<link>`
- todo not needed, we can log on the server
- removed the `</body>` tag because A) the browser will figure it out, B) [`ViewAsRenderMiddleware` ](https://github.com/getsentry/getsentry/blob/4138852300adce937b1cbea9d727b2391ee8aef1/getsentry/vendor/viewas/middleware.py#L102-L132) was injecting some extra html into there, which wasn't breaking anything afaik, but it also was just adding weight to a iframe that no one will see. 
- Also, i added a conditional for the logging stuff. It can be noisy to have logging on by default, but nice to have when we need it.

### CODE REVIEW

**Review by c298lee — APPROVED** [review]

File: src/sentry/templates/sentry/toolbar/iframe-invalid.html [hunk 1]
```diff
@@ -35,5 +34,8 @@
       {% endif %}
     </script>
     {% endscript %}
-  </body>
-</html>
+
+{% comment %}
+No need to close `body`. If we do then middleware will inject some extra markup
```
Comment (aliu39) [inline src/sentry/templates/sentry/toolbar/iframe-invalid.html:39]: what about closing <html>?

Comment (ryan953) [reply src/sentry/templates/sentry/toolbar/iframe-invalid.html:39]: the browser will sort it out too

File: src/sentry/templates/sentry/toolbar/login-success.html [hunk 2]
```diff
@@ -15,28 +15,23 @@
...
+        window.close();
+      });
 
-          window.opener.postMessage({
-            source: 'sentry-toolbar',
-            message: 'did-login',
-          }, '*');
-        }
+      if (!window.opener) {
+        return;
```
Comment (aliu39) [inline src/sentry/templates/sentry/toolbar/login-success.html:24]: does `return` work at the top level of a <script> ? TIL

**Review by aliu39 — APPROVED** [review]

File: src/sentry/templates/sentry/toolbar/login-success.html (hunk 2 above)
Comment (ryan953) [reply src/sentry/templates/sentry/toolbar/login-success.html:24]: ya, it just bails

Comment (ryan953) [reply src/sentry/templates/sentry/toolbar/login-success.html:24]: ah, i lied it complains

**Pull request merged**